cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -248,6 +248,21 @@ def after_fill(self, was_change):
             wait_for(lambda: not self.save.disabled, timeout='10s', delay=0.2)
 
 
+class CopyCatalogItemView(ServicesCatalogView):
+    name = Input(name="name")
+
+    add = Button("Add")
+    cancel = Button("Cancel")
```

Comment:
**Optional:**
Make `button` variable/object names more readable like:
`add_btn` and `cancel_btn`

Reply:
This is nice point :+1:..
here i fallowed older implementation of file; If you check `catalog_items/__init__.py` mostly we used `add` or `cancel` not adding extension like `btn`. I think both notation are good, but try to fallow single in model page (means sure about consistency for specific model page).

Reply:
I wouldn't change button names. we already follow some approach. code should look similar everywhere